charts/apitestrig/values.yaml

```diff
@@ -488,3 +488,22 @@ modules:
     enabled: true
 
 crontime: "0 3 * * *"    ## run cronjob every day at 3 AM (time hr: 0-23 )
+
+apitestrig:
+  configmaps:
+    s3:
+      s3-host: 'http://minio.minio:9000'
+      s3-user-key: 'admin'
+      s3-region: ''
+    db:
+      db-port: '5432'
+      db-su-user: 'postgres'
+      db-server: 'postgres.sandbox.xyz.net'
+    apitestrig:
+      ENV_USER: 'api-internal.sandbox'
+      ENV_ENDPOINT: 'https://api-internal.sandbox.xyz.net'
+      ENV_TESTLEVEL: 'smokeAndRegression'
+      ENABLE_INSECURE: 'false'
+  secrets:
+    db:
+      db-su-password: ''
```

Comment:
since we already are using db password as db-common-password, lets do same here and copy the same secret here.
Lets change the variable in apitest rig as required.